frontend/src/queries/nodes/DataVisualization/Components/DisplayTab.tsx

```diff
@@ -84,96 +86,119 @@ export const DisplayTab = (): JSX.Element => {
-                                    removeGoalLine={removeGoalLine}
-                                    updateGoalLine={updateGoalLine}
-                                />
-                                <LemonButton className="mt-1" onClick={addGoalLine} icon={<IconPlusSmall />} fullWidth>
-                                    Add goal line
-                                </LemonButton>
+                                {!isTotalValueChart && (
+                                    <>
+                                        <LemonSwitch
+                                            className="flex-1 w-full"
+                                            label="Show total row"
+                                            checked={chartSettings.showTotalRow ?? true}
+                                            onChange={(value) => {
+                                                updateChartSettings({ showTotalRow: value })
+                                            }}
+                                        />
+                                        <LemonSwitch
+                                            className="flex-1 w-full"
+                                            label="Show X-axis labels"
+                                            checked={chartSettings.showXAxisTicks ?? true}
+                                            onChange={(value) => {
+                                                updateChartSettings({ showXAxisTicks: value })
+                                            }}
+                                        />
+                                        <LemonSwitch
+                                            className="flex-1 w-full"
+                                            label="Show X-axis border"
+                                            checked={chartSettings.showXAxisBorder ?? true}
+                                            onChange={(value) => {
+                                                updateChartSettings({ showXAxisBorder: value })
+                                            }}
+                                        />
+                                        <LemonSwitch
+                                            className="flex-1 w-full"
+                                            label="Show Y-axis border"
+                                            checked={chartSettings.showYAxisBorder ?? true}
+                                            onChange={(value) => {
+                                                updateChartSettings({ showYAxisBorder: value })
+                                            }}
+                                        />
+                                    </>
+                                )}
                             </>
                         ),
                     },
+                    ...(isTotalValueChart
+                        ? []
+                        : [
+                              {
```

Comment:
Pet peeve, but I despise these inline ternaries, makes it so hard to actually read what's going on in the code. Tbh its probably time to abstract this out to come controller, like `chart x gets settings y & z`